lib/handlers/quote/util/simulation.ts

```diff
@@ -1,16 +1,20 @@
-import { SimulationStatus } from "@uniswap/smart-order-router"
-import Logger from "bunyan"
+import { SimulationStatus } from '@uniswap/smart-order-router'
+import Logger from 'bunyan'
 
 export const simulationStatusToString = (simulationStatus: SimulationStatus, log: Logger) => {
   switch (simulationStatus) {
     case SimulationStatus.Succeeded:
       return 'SUCCESS'
     case SimulationStatus.Failed:
       return 'FAILED'
-    case SimulationStatus.Unattempted:
-      return 'UNATTEMPTED'
+    case SimulationStatus.NotApproved:
+      return 'NOTAPPROVED'
+    case SimulationStatus.InsufficientBalance:
+      return 'INSUFFICIENTBALANCE'
+    case SimulationStatus.NotSupported:
+      return 'NOTSUPPORTED'
```

Comment:
Nit: would it make sense for these enums to be snake case? Feel like that's a pattern I've seen us use elsewhere. 
```suggestion
    case SimulationStatus.NotApproved:
      return 'NOT_APPROVED'
    case SimulationStatus.InsufficientBalance:
      return 'INSUFFICIENT_BALANCE'
    case SimulationStatus.NotSupported:
      return 'NOT_SUPPORTED'
```